hooks/useRailgunTx.tsx

```diff
@@ -140,24 +86,23 @@ const useRailgunTx = () => {
 
     // Public wallet to shield from.
     transactionRequest.from = address;
+    transactionRequest.gasPrice = undefined;
+    transactionRequest.gasLimit = undefined;
```

Comment:
gasPrice and gasLimit are set to a BigNumber of 0 which was preventing sendTransaction from overriding these values. Probably good feedback for the Railgun team which I can send over tomorrow.

Reply:
IIUC the default `transactionRequest` we get back with the new SDK version has them set to zero, so wallets don't do the estimation, and we fix that be re-setting them to `undefined` here? If so, makes sense. Let's leave this unresolved until we get that feedback over to them so we don't forget

Reply:
Yes, exactly

Reply:
Saw you sent it, so marking resolved